iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/ReferenceTerminals.java

```diff
@@ -70,15 +74,25 @@ static void addTerminal(Terminal terminal) {
     }
 
     /**
-     * Gets the reference terminals in the network for the current variant
+     * Gets the reference terminals in the network for the current variant.
+     * In case of a merged network with subnetworks, reference terminals from the merged network
+     * and reference terminals from subnetworks are returned altogether.
```

Comment:
this is what eg CGMES SV exporter will want to do

Reply:
Not sure about this. IGM SV export: the extension from the subnetwork, CGM SV export, the extension from the merged network only.

Reply:
I reverted this change